zilliqa/src/db.rs

```diff
@@ -253,6 +253,8 @@ impl Db {
                 PRIMARY KEY (address, tx_hash));
             CREATE TABLE IF NOT EXISTS tip_info (
                 latest_finalized_view INTEGER,
+                latest_view INTEGER,
```

Comment:
I wonder if `latest_view` might be a slightly misleading name. It looks very similar to `latest_finalized_view` which is pointing to our latest finalized block, however `latest_view` won't ever point to an actual block (by design).

I can't think of anything good though, other than just `view`?

Reply:
Agreed, updated to `view`